cf-oscore/src/main/java/org/eclipse/californium/oscore/OSCoreCtx.java

```diff
@@ -834,7 +842,11 @@ public synchronized void rollBack() {
 	private static int ivLength(AlgorithmID alg) {
```

Comment:
I would prefer, if this is moved to "EncryptCommon" (use the finals there) and made public.

Reply:
Yes, that makes sense. This has now been done in commit 8c0bee8df9e1eb0fb862725a36d47c2ed6f4965a